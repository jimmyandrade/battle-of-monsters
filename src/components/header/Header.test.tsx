import { Header } from '.'
import { render } from '@testing-library/react'

describe('Header component', () => {
  it('should match snapshot', () => {
    const { container } = render(<Header />)
    expect(container).toMatchSnapshot()
  })
})
