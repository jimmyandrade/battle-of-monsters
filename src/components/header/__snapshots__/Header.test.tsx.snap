// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
.c1 {
  margin: 0 auto;
  width: 814px;
}

.c0 {
  padding-block-end: 35px;
  padding-block-start: 69px;
}

<div>
  <header
    class="c0"
  >
    <div
      class="c1"
    >
      <h1>
        Battle of Monsters
      </h1>
    </div>
  </header>
</div>
`;
